include/fmt/ranges.h

```diff
@@ -414,20 +417,86 @@ struct range_formatter<
     if (it != end && *it == 'n') {
       set_brackets({}, {});
       ++it;
+    } else {
```

Comment:
According to https://eel.is/c++draft/format#range.formatter-2 `n` and other specifiers are not mutually exclusive and therefore `else` is not needed here.

Reply:
But below that, it notes "[If the range-type is s or ?s, then there shall be no n option and no range-underlying-spec.](https://eel.is/c++draft/format#range.formatter-6.sentence-3)".

Reply:
Ah, missed that. Makes sense now.